### ISSUE TITLE
ImportError: libcublas.so.8.0

### ISSUE THREAD

**Issue by roby23** [JesperChristensen89/TensorFlow-Jetson-TX2#2]
I'm trying to install tensorflow-1.5 on a Jetson TX2. 
The installation goes well, but when I import tensorflow in python I get the following error:

ImportError: libcublas.so.8.0: cannot open shared object file: No such file or directory

I have CUDA 9.0 on my system (libcublas.so.9.0) and on your guide you say that you built it for CUDA 9.0 and cuDNN 7.0.5 RC for Jetson TX2

Any help on this?

Many thanks

**Comment by DavidTorresOcana** [comment]
Make sure you have exported environment variables:

`export LD_LIBRARY_PATH=/usr/local/cuda-9.0/lib64`
`export PATH="$PATH:/usr/local/cuda-9.0/bin"`

**Comment by roby23** [comment]
Yes, I have exported it, but I have libcublas.so.9.0 int those folders, not libcublas.so.8.0

**Comment by DavidTorresOcana** [comment]
I would assume you removed CUDA 8 and that any cuda symlink points to the right folder.

Remember to add the above commands to ~/.bashrc
Do you have Python3/pip3 in your system? 

As a side comment, see this threads. They helped me to install TF1.7 with CUDA 9 in DPX2:
[https://devtalk.nvidia.com/default/topic/998005/driveworks/installation-of-tensorflow-on-drive-px/2#reply](url)
[https://devtalk.nvidia.com/default/topic/1031300/jetson-tx2/tensorflow-1-7-wheel-with-jetpack-3-2-](url)

**Comment by arthurherbout** [comment]
Hi, 
I have the exact same problem, no reason that tensorflow asks for  libcublas.so.8.0.
Any way out of this exempt from downloading cuda 8 ?

**Comment by tusharc7360** [comment]
I am facing the same issue.
Any solution for this?
